app/lib/dfe/bigquery/test_helper.rb

```diff
@@ -0,0 +1,45 @@
+module DfE
```

Comment:
I decided to create this module into app/lib because I think it will be easier when we extract the app/lib/dfe/bigquery to a gem. Also my gut feeling say here was the best place to add.